mail_debrand/models/mail_template.py

```diff
@@ -10,13 +10,23 @@ class MailTemplate(models.Model):
     _inherit = "mail.template"
 
     @api.model
-    def _debrand_body(self, html):
-        using_word = _('using')
-        odoo_word = _('Odoo')
-        html = re.sub(
-            using_word + "(.*)[\r\n]*(.*)>" + odoo_word + r"</a>", "", html,
-        )
-        powered_by = _("Powered by")
+    def _debrand_translated_words(self, lang):
```

Comment:
Could you please check why here we don't have anymore the copyright edits that were in https://github.com/OCA/social/pull/613?
The same happens for the author field in manifest

Reply:
This can be resolved?